src/ducks/transactions/scroll/InfiniteScroll.jsx

```diff
@@ -24,7 +24,7 @@ class InfiniteScroll extends React.Component {
   componentDidMount() {
     this.scrollingElement = this.getScrollingElement()
     this.listenToScroll()
-    this.checkForLimits()
+    setTimeout(() => this.checkForLimits(), 0)
```

Comment:
I didn't take a look at the `onReachTop` props for exemple, but are you doing some animation? Using `requestAnimationFrame` is not enough to put an event in the loop right after the render execution?

Reply:
I went another way eventually (see https://github.com/cozy/cozy-banks/pull/1646#discussion_r341947599), but yes, requestAnimationFrame could also work.